src/core/util/featureMap.js

```diff
@@ -8,20 +8,39 @@
  * SPDX-License-Identifier: MPL-2.0
  ****************************************************************************/
 
-// Stores entries in following hierarchical order
-// <type>.<feature>.<instanceId>.{ $metadata, }
 
-module.exports = class FeatureMap {
```

Comment:
Is this export not needed anymore?

Reply:
Ah sorry, I haven't seen that these exports are moved to the bottom.
Is there are reason for that? (For me both is ok)